adapter/internal/oasparser/model/http_route.go

```diff
@@ -308,6 +343,16 @@ func concatAuthSchemes(schemeUp *dpv1alpha1.Authentication, schemeDown *dpv1alph
 	return &finalAuth
 }
 
+func concatRateLimitPolicies(schemeUp *dpv1alpha1.RateLimitPolicy, schemeDown *dpv1alpha1.RateLimitPolicy) *dpv1alpha1.RateLimitPolicy {
+	finalRateLimit := dpv1alpha1.RateLimitPolicy{}
+	if schemeUp != nil && schemeDown != nil {
+		finalRateLimit.Spec.Override = *utils.SelectPolicy(&schemeUp.Spec.Override, &schemeUp.Spec.Default, &schemeDown.Spec.Override, &schemeDown.Spec.Default)
+	} else if schemeUp != nil {
+		finalRateLimit.Spec.Override = *utils.SelectPolicy(&schemeUp.Spec.Override, &schemeUp.Spec.Default, nil, nil)
+	}
```

Comment:
don't we need another case when only the  `schemeDown` is available?

Reply:
the usages of this util didn't have that case yet, so haven't added it.